webapp/src/js/services/db-sync.js

```diff
@@ -4,7 +4,8 @@ const DDOC_PREFIX = ['_design/'];
 const LAST_REPLICATED_SEQ_KEY = 'medic-last-replicated-seq';
 const LAST_REPLICATED_DATE_KEY = 'medic-last-replicated-date';
 const SYNC_INTERVAL = 5 * 60 * 1000; // 5 minutes
-const META_SYNC_INTERVAL = 30 * 60 * 1000; // 30 minutes
+//const META_SYNC_INTERVAL = 30 * 60 * 1000; // 30 minutes
+const META_SYNC_INTERVAL = 30  * 1000; // 30 seconds
```

Comment:
Oops!

Reply:
Whoops. Serves me right in rushing this to get reviewed :(